configurations/object_storage_rclone_method.yaml

```diff
@@ -0,0 +1 @@
+object_storage_method: "rclone"
```

Comment:
@pehala please advise, this file is not currently in use, following  the assumption rClone is not regularly tested.
So it's possible to completely remove this file and only override this parameter's env var when needed.

Reply:
Added a new pipeline:
`jenkins-pipelines/manager/restore_benchmark/ubuntu22-manager-1tb-2t-9nodes-rclone.jenkinsfile`